dat/data.base

```diff
@@ -2234,6 +2363,11 @@ hell hound*
 	dark form and savage face which broke upon us out of the wall
 	of fog.
 	  [ The Hound of the Baskervilles, by Sir Arthur Conan Doyle ]
+helm of brilliance
```

Comment:
I don't understand what about the quote is specific to a helm of brilliance, unless it's "brilliance" in the sense of "radiant beauty" (which I don't think is how it's used in the item name).  This seems to me like it would be a better quote for "helmet" in general.